google/cloud/compute/v1/compute_small.proto

```diff
@@ -686,6 +701,7 @@ service RegionOperations {
       get: "/compute/v1/projects/{project}/regions/{region}/operations/{operation}"
     };
     option (google.api.method_signature) = "project,region,operation";
+    option (operation_polling_method) = true;
```

Comment:
And shouldn't there also be an `operation_cancel_method` and `operation_delete_method` annotation? How else will we know that the `Delete` method is used for this purpose?

Reply:
- the `operation_polling_method` it's not in the design doc, but it's in the new annotations: https://source.corp.google.com/piper///depot/google3/third_party/googleapis/google/cloud/extended_operations.proto;rcl=392508615;l=120
- there seems to be no need for `operation_cancel_method` since AFAIK Gapic LRO wrappers do not provide cancel of delete functionality, only 'poll until completion'

Reply:
Thanks for the clarification! I suggest we do mention `operation_polling_method` in the design doc, and also point from there to the checked-in `extended_operations.proto`